app/controllers/concerns/accepts_jwt.rb

```diff
@@ -0,0 +1,17 @@
+module AcceptsJwt
+  extend ActiveSupport::Concern
+
+  def find_or_create_jwt(payload = nil)
+    @find_or_create_jwt ||=
+      begin
+        if payload
+          Jwt.create!(jwt_payload: payload).tap { |j| session[:jwt_id] = j.id }
+        elsif session[:jwt_id]
+          Jwt.find(session[:jwt_id])
+        end
+      rescue ActiveRecord::RecordNotFound
+        session.delete(:jwt_id)
+        nil
+      end
```

Comment:
Could we use `find_by` that returns `nil` if no record is found instead using `find` and rescuing the exception?